src/main/java/course_project/Main.java

```diff
@@ -0,0 +1,12 @@
+package course_project;
+
+import course_project.Utils.ConsoleHandling.Printer;
+import course_project.components.Field;
+
+public class Main {
```

Comment:
![image](https://user-images.githubusercontent.com/78850141/134902182-b5ede70d-e576-4726-8d54-3d0f282529c9.png)
The computer shot it shown only on the next computer turn

Reply:
Opt: cells around a destroyed ship could be marked as miss automatically
![Uploading image.png…]()

Reply:
Opt: cells around a destroyed ship could be marked as miss automatically
![image](https://user-images.githubusercontent.com/78850141/134902429-ce2958fe-16d6-4c51-9d5f-137b6c089c65.png)

Reply:
Well-done! The game is confortable to play, has nice interface, automatic ship placement, informative messages. The visual representation is also good.

Reply:
From the code perspective also looks good, there are minor issues but in general, the code is readable, easy to follow, has nice abstractions. You show good knowledge of Java Core and Java libraries. Approved!